src/WebJobs.Extensions.DurableTask/Listener/DurableTaskTriggerMetrics.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#if NETSTANDARD2_0
+using System;
+using System.Collections.Generic;
+using System.Text;
+using DurableTask.AzureStorage.Monitoring;
+using Microsoft.Azure.WebJobs.Host.Scale;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class DurableTaskTriggerMetrics : ScaleMetrics
+    {
+        /// <summary>
+        /// The number of partitions in the task hub.
+        /// </summary>
+        public int PartitionCount { get; set; }
+
+        /// <summary>
+        /// The number of messages across control queues.

```

Comment:
What will be the format of this. Can you include an example in a remarks section please? For the latencies property as well